src/client.rs

```diff
@@ -526,9 +570,18 @@ fn main() {
             Event::UserInputEvent(user_input)
         }).map_err(|_| ());
 
+    let network_stream = stream::iter_ok::<_, io::Error>(iter::repeat( () ));
+    let network_stream = network_stream.and_then(|_| {
```

Comment:
Why not `tick_stream` and `Event::TickEvent`?

Reply:
I wanted to be able to send the keep-alives at a different rate than all of the other TX network traffic. Currently they're both set to run every second but I don't think those are the right values.